components/eam/src/physics/crm/samxx/test/cpp_driver.F90

```diff
@@ -88,22 +89,12 @@ program driver
   endif
 
   ! Allocate model data
-  call crm_input%initialize (           ncrms,plev)
-  call crm_output_initialize(crm_output,ncrms,plev)
+  call crm_state_initialize(crm_state  , ncrms, crm_nx, crm_ny, crm_nz, trim(microphysics_scheme))
+  call crm_rad_initialize  (crm_rad    , ncrms, crm_nx_rad, crm_ny_rad, crm_nz, trim(microphysics_scheme))
+  call crm_input_initialize(crm_input  , ncrms, plev, trim(microphysics_scheme))
+  call crm_output_initialize(crm_output, ncrms, plev, crm_nx, crm_ny, crm_nz, trim(microphysics_scheme))
```

Comment:
Why do you need the "trim" here? don't the init routines apply the trim? I realize it doesn't hurt, just wondering if I'm missing something.